src/utils/windows.js

```diff
@@ -59,6 +59,7 @@ export const createAttributes = attrs => ({
   shadowDOM: false,
   clamp: true,
   droppable: true,
+  droppableDataTransferProperty: 'files',
```

Comment:
Could `droppable` be `boolean | { droppableDataTransferProperty?: 'files' | 'items' }` instead here maybe ?

Reply:
And then rename to `dataTransferProperty`

Reply:
I agree. Done 👍